src/pages/ProjectsPage.js

```diff
@@ -1,126 +1,146 @@
-                      <Button className="urgent-tag"> Urgent </Button>
-                      <Text>
-                        Project description. We are looking for a group of people to help us put together a simple
-                        website for people to email their state reps.
-                      </Text>
-                      <Link href="#" className="float-right" onClick={() => this.setState({ selected: project.id })}>
-                        More details...
-                      </Link>
-                    </Body>
-                  </Card>
-                ))}
-              </Col>
+const { Body, Subtitle } = Card;
 
-              <Col lg={7} md={12} className="">
-                <Card className="description-card">
-                  <Body>
-                    <div class="d-flex justify-content-lg-end justify-content-md-start">
-                      <Button className="close-btn mb-4 mb-lg-0" onClick={() => this.setState({ selected: -1 })}>
-                        Close
-                      </Button>
-                    </div>
-
-                    <h4> {projects[selected].title}</h4>
-                    <Subtitle className="mb-2 text-muted">{projects[selected].company}</Subtitle>
-                    <Button className="urgent-tag"> Urgent </Button>
-
-                    <h5 className="pt-3"> About {projects[selected].company} </h5>
-                    <p>
-                      This is a more detailed example about what a card could say. It would have information here about
-                      what the company does and how people can get involved.
-                    </p>
-
-                    <h5 className="pt-3"> About the Project </h5>
-                    <p>
-                      This is a more detailed example about what a card could say. It would have information here about
-                      what the company does and how people can get involved.
-                    </p>
-
-                    <h5 className="pt-3"> Project Timeline </h5>
-                    <p class="card-text">Estimated 2 weeks - Need ASAP </p>
-
-                    <div class="text-center">
-                      <LinkContainer to={`/projects/${selected}`}>
-                        <Button className="primary-button"> Work on this project! </Button>
-                      </LinkContainer>
-                    </div>
-                  </Body>
-                </Card>
-              </Col>
-            </Row>
-          </Container>
-        )}
-      </>
-    );
+// example projects, delete this later
+const example = (num) => ({
+  id: num,
+  isUrgent: num %2 === 0,
+  orgName: 'HackBeanpot ' + num,
+  orgAbout:
+    'this is what our org does. Which likely has a lot of words, resulting in multiple lines on the screen. We love beans in our org. bean bean bean bean bean bean bean bean bean bean bean bean bean bean bean bean bean bean',
+  orgEmail: 'team@hbp.com',
+  orgPhone: null,
```

Comment:
this is not used anywhere in this code which is fine, but we can delete "orgPhone" since we're not collecting this info in the request form